Shakikka/refactorTractor
# Refactor/user class

## What does this PR do?

- Refactor User tests related to iteration 1 of original FitLit specs. Updates include establishing exports/imports of test data from fakeData.js, implementing beforeEach to DRY up code, and adding tests to cover every property in User.

## How should it be tested?

- Review the tests in user-test.js and ensure they accurately test the User properties. Then run the tests and all should pass.
